frontend/src/modules/dashboard/containers/ControlInfo.tsx

```diff
@@ -115,30 +154,55 @@ export const ValueControl = ({ selector, label, units }: ValueProps): JSX.Elemen
 };
 
 /**
- * Control Info
+ * ControlInfo
  *
- * Component for showing information.
+ * @component UI wrapper to display Value & Set Value Modal popup
+ *
+ * Uses the [[Props]] interface
+ *
+ * @returns {JSX.Element}
  *
  */
 const ControlInfo = (props: Props): JSX.Element => {
   const { selector, label, units, stateKey, committedSettingSelector, min, max } = props;
   const [open, setOpen] = useState(false);
   const committedSetting = useSelector(committedSettingSelector);
   const dispatch = useDispatch();
+
+  /**
+   * Update paramter value to redux store
+   *
+   * @param {number} setting value
+   *
+   */
   const doSetValue = (setting: number) => {
     const update = { [stateKey]: setting };
     dispatch(commitRequest<ParametersRequest>(MessageType.ParametersRequest, update));
     dispatch(commitDraftRequest<ParametersRequest>(MessageType.ParametersRequest, update));
   };
+
+  /**
+   * Opens Multistep Popup on Clicking over component
+   */
   const onClick = () => {
     // setOpen(true);
     if (stateKey) {
       setMultiPopupOpen(true, stateKey);
     }
   };
+
+  /**
+   * Disable click events over component
+   */
   const handleClick = ClickHandler(onClick, () => {
     return false;
   });
+
+  /**
+   * Function for updating the modal status.
```

Comment:
Yes we can remove it. We just added as a wrapper function if in case we need to do some processing. In this case, `setOpen` can replace `updateModalStatus`